### ISSUE TITLE
Play speed change button

### ISSUE THREAD

**Issue by SuperDZ555** [Sequitur-Studios/Cell-Machine-Mystic-Mod#109]
**Is your feature request related to a problem? Please describe.**
I'm tired of trying to see what would happen if something happens, but to do that i would have to wait an extremely long amount of time.

**Describe the solution you'd like**
A way to change the play speed, possibly using a slider that appears after you press the "play" button

**Describe alternatives you've considered**
I cant really think of any other options besides putting the slider in the pause menu

**Additional context**
Here's what I mean: So, imagine there is a Mover Cell heading towards an Enemy Cell, with the distance between the two cells being just right such that it would take the Mover Cell exactly one minute to get to the Enemy Cell. But some people wouldn't want to wait that long. So, my idea, is to add a little slider at the bottom of the screen that can change the speed of everything, but in a way that wouldn't really cause any cell functionality to change. For example, the functionality of the Mover Cell would stay the same, with it being moving forward one space per step, although the steps themselves would be faster. Back to our original scenario with the Mover Cell going towards the Enemy Cell with the collision taking 1 minute, keeping the slider at 1x would keep it at 1 minute, but increasing the slider to 2x would set it to 30 seconds, and then 3x being 20 seconds, i think, and then 4x being 15 seconds, and so on, you get the point. I'm sorry if that was hard to follow, I couldn't really find any better way of explaining it.

**Comment by w1wwwwww** [comment]
already exists. press ~ to open the menu

**Comment by blaumeise20** [comment]
Weird. Some people say you have to press <code>`</code>. I would really like to have that slider.

**Comment by uku1928** [comment]
I really hope this feature gets out of debug

**Comment by TheMysticLynx** [comment]
It probably should be moved from the debug menu to in game menu.
